apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -60,6 +61,21 @@ function register_blocks() {
 			'render_callback' => 'render_template_part_block',
 		) );
 	}
+
+	/**
+	 * This will set the `wp_template` post type to `public` to support
+	 * the core search endpoint, which looks for it.
+	 *
+	 * @return void
+	 */
+	function allow_searching_for_templates() {
+		$post_type = get_post_type_object( 'wp_template' );
+		if ( ! is_a( $post_type, 'WP_Post_Type' ) ) {
```

Comment:
Do you think using `instanceof` would be an option here? Core usually prefers that over `is_a()`

Reply:
that's awesome to know @obenland - is there an explanation why? the PHP docs weren't clear to me what the differences in behaviors are.

Reply:
I first thought because it was slightly faster than calling a function, the actual reason was a different one: https://core.trac.wordpress.org/changeset/31188

Reply:
Thanks for the extra knowledge @obenland - I just assumed this was yet another place where PHP has multiple ways to do the same thing.